aws-sdk/aws-sdk.d.ts

```diff
@@ -1072,7 +1072,7 @@ declare module "aws-sdk" {
 			GrantReadACP?: string;
 			GrantWriteACP?: string;
 			Key: string;
-			Metadata?: string[];
+			Metadata?: any;
```

Comment:
@vvakame Thanks for your quick review! Didn't know you could that. Just updated the code accordingly.